specs/AppLifecycle/Restart/restartApi.markdown

```diff
@@ -0,0 +1,357 @@
+# Reunion Restart API Spec
+
+## Contents
+
+[1. Overview](#overview)
+- [Reunion Restart API components](#reunion-restart-api-components)
+
+[2. API](#api)
+
+- [Existing API Summary](#existing-api-summary)
+
+- [All Existing Scenarios](#all-existing-scenarios)
+
+- [Existing API Details](#existing-api-details)
+
+- [New API Details](#new-api-details)
+
+- [API Sample Code](#api-sample-code)
+
+[3. Telemetry](##restartApi\telemetry)
+
+[4. Resources](#resources)
```

Comment:
I think your TOC is out of sync with your actual content

SUGGESTION: Use VSCode with the [Marodnw All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) extension, then issue commands via Ctrl-P

1. Markdown All in One: Add/Update section numbers
2. Put cursor at top of file
3. Markdown All in One: Create Table of Contents

and if/when you make edits the pair of ...Add/Update section numbers + ...Update Table of Contents keeps everything correct and current